xml/System.Security.Cryptography.Cose/CoseHeaderMap.xml

```diff
@@ -322,6 +323,7 @@
         <returns>
           <see langword="true" /> if the <see cref="T:System.Collections.Generic.IDictionary`2" /> contains an element with the key; otherwise, <see langword="false" />.</returns>
         <remarks>To be added.</remarks>
+        <inheritdoc />
```

Comment:
@Jozkee this type inherits from two different interfaces, making the inheritdoc tag ambiguous without a cref, and mdoc is unable to decide where to get he documentation from:

```xml
      <Implements>
        <InterfaceMember>M:System.Collections.Generic.IDictionary`2.ContainsKey(`0)</InterfaceMember>
        <InterfaceMember>M:System.Collections.Generic.IReadOnlyDictionary`2.ContainsKey(`0)</InterfaceMember>
      </Implements>
```

Do you have a preference? Change it if you prefer the other one:
```suggestion
        <inheritdoc cref="M:System.Collections.Generic.IDictionary`2.ContainsKey(`0)" />
```